en/02_Developer_Guides/02_Controllers/01_Introduction.md

```diff
@@ -60,16 +55,18 @@ SilverStripe\Control\Director:
     'teams//$Action/$ID/$Name': 'App\Control\TeamController'
 ```
 
+> [!CAUTION]
+> Make sure that after you have modified the `routes.yml` file, that you clear your Silverstripe CMS caches using `?flush=1`.
```

Comment:
Moved this from above to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.